src/constants/event-emitter.ts

```diff
@@ -22,6 +22,9 @@
  * SOFTWARE.                                                                      *
  **********************************************************************************/
 
-export const PROBE_RESPONSE_RECEIVED = 'PROBE_RESPONSE_RECEIVED'
-export const PROBE_RESPONSE_VALIDATED = 'PROBE_RESPONSE_VALIDATED'
-export const PROBE_STATUS_PROCESSED = 'PROBE_STATUS_PROCESSED'
+export const RESPONSES_READY_TO_PROCESS = 'RESPONSES_READY_TO_PROCESS' // 1. probe done, process responses
+export const PROBE_RESPONSE_VALIDATED = 'PROBE_RESPONSE_VALIDATED' // 2. results validated, good to process
+
+export const ALERTS_READY_TO_SEND = 'ALERTS_READY_TO_SEND' // 3. alerts/notifications built and ready to send
```

Comment:
I think the previous name `PROBE_STATUS_PROCESSED` is better as an event name, because it is more about probe, and it can be listened by anything or by any plugin.

Perhaps in the future the event will be used not only by alerting system.

Reply:
ok noted. using a more generic name.